### PR TITLE
modify the error url about setup-csi-snapshotter

### PR DESCRIPTION
**Pull request by timyinshi**
the error url is: https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes/setup-csi-snapshotter.yaml
modify to: https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml

/kind bug
/sig docs

### CODE REVIEW

File: book/src/external-snapshotter.md [hunk 1]
```diff
@@ -34,4 +34,4 @@ For detailed information (binary parameters, RBAC rules, etc.), see [https://git
 
 The CSI `external-snapshotter` is deployed as a controller. See [deployment section](deploying.md) for more details.
 
-For an example deployment, see [this example](https://github.com/kubernetes-csi/external-snapshotter/tree/master/deploy/kubernetes/setup-csi-snapshotter.yaml) which deploys `external-snapshotter` and `external-provisioner` with the Hostpath CSI driver.
+For an example deployment, see [this example](https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/csi-snapshotter/setup-csi-snapshotter.yaml) which deploys `external-snapshotter` and `external-provisioner` with the Hostpath CSI driver.
```
Comment (xing-yang) [inline book/src/external-snapshotter.md]: Use this one: https://github.com/kubernetes-csi/external-snapshotter/blob/release-1.2/deploy/kubernetes/setup-csi-snapshotter.yaml

We need to update the doc soon with information on how to deploy the beta version.  For now the latest correct one is in release-1.2 which deploys the alpha version of the snapshot sidecar.

Comment (timyinshi) [reply book/src/external-snapshotter.md]: ok,i modify the url for release-1.2

**Comment by msau42** [comment]
Thanks for the update! Can you also squash your commits?

/release-note-none
/approve

**Pull request merged**